Add warning to LDAP settings for AD requirements

michaeltlombardi opened the pull request:
When setting up the LDAP integration to Active Directory, certain settings are *required*.
The documentation now includes clear instructions areound which settings are absolutely
required for the binding.

mabrahms-chef commented:
Hi username-is-already-taken2. I've incorporated your feedback. There's a wrinkle in accepting PRs which we are working out. In the meantime, please keep submitting PRs on the docs and James or I will incorporate them, or let you know when we can start accepting PRs directly. Thanks for the feedback, and sorry for the delay on getting it into the doc. Go to https://docs.chef.io/windows.html to see your change.

michaeltlombardi committed:
Add warning to LDAP settings for AD requirements

When setting up the LDAP integration to Active Directory, certain settings are *required*.
The documentation now includes clear instructions areound which settings are absolutely
required for the binding.

michaeltlombardi commented:
I discovered this by trying to bind without a `group_dn`. 😭

kagarmoe commented:
![tenor-158248419](https://cloud.githubusercontent.com/assets/4400151/26015359/886d6254-3715-11e7-99fa-1d82d06f6faf.gif)
